src/components/TopBar/CallButton.vue

```diff
@@ -14,6 +14,7 @@
 				autoHide: false,
 				html: true
 			}"
+			:aria-label="isMobile ? startCallLabel : undefined"
```

Comment:
```suggestion
			:aria-label="startCallLabel"
```

a11y team passed by and said "Both visible label and aria-label could be used together, if they have the save content"

Reply:
done